src/rdds/lib/vcf/parsable_variant.py

```diff
@@ -20,42 +51,35 @@ def __init__(self,
         :param parse_only_fields: List of INFO fields to parse
         :param vep_csq_description: CSQ.DESCRIPTION field containing 'Format: 'Allele|Consequence|...' string
 
-        TODO: Support for FORMAT fields
+        TODO: Parse FORMAT fields
         """
         if not isinstance(variant, Variant):
             raise TypeError(f'Expected cyvcf2.Variant got {type(self)}')
 
-        self.chrom: str = str(variant.CHROM)  # chrom idx + x, y, mit as char
-        self.pos: int = int(variant.POS)  # integer
-        self.id: str = str(variant.ID)  # text field
-        # TODO: Parsing of REF/ALT fields with <ID> subtype, see https://samtools.github.io/hts-specs/VCFv4.1.pdf, p4
-        self.ref: str = str(variant.REF).replace('[', '').replace(']', '')
-        self.alt: str = str(variant.ALT).replace('[', '').replace(']', '')
-
         self._vep_csq_description: str = vep_csq_description
 
-        self._parsed_fields: List[str] = []
-
-        if parse_only_fields is None:
-            parse_only_fields: List[str] = [key for key, value in list(variant.INFO)]
-
         # Store all meta data as private instance attributes
-        for key, value in variant.INFO:
+        self.CHROM: str = str(variant.CHROM)  # chrom idx + x, y, mit as char
+        self.POS: int = int(variant.POS)  # integer
+        self.ID: str = str(variant.ID)  # text field
```

Comment:
```suggestion
        self.CHROM = str(variant.CHROM)
        self.POS = int(variant.POS)
        self.ID: = str(variant.ID) 
```
I do not think you need to type hint when calling a constructor. This is clear enough.